### PR TITLE
Setup generic components from MaterialUI

### LINKED ISSUE #5: Create generic React Components.

### PR DESCRIPTION
**Pull request by nitishmittal1990**
Implemented Following Generic Components:
- [X] Theme Setup
- [X] Button
- [X] Box
- [X] Checkbox
- [X] Grid
- [X] Loader
- [X] Modal
- [X] TextField

Closes #5

### CODE REVIEW

**Comment by fenilgandhi** [comment]
can you edit the title ?

File: client/src/components/app/App.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 import React from 'react';
+
 import style from './App.module.css';
 
 function App() {
```
Comment (nimish-gupta) [inline client/src/components/app/App.js]: Please use the ES6 syntax here. For example- this function can be changed to 
```suggestion
const App = <div className={style.App}>...</div>;
```

**Review by nimish-gupta — APPROVED** [review]
LGTM, just add a screenshot in every front end PR.

File: client/src/components/app/App.js (hunk 1 above)
Comment (nitishmittal1990) [reply client/src/components/app/App.js]: Updated

**Review by vineetvk01 — APPROVED** [review]
LGTM

**Pull request merged**